File: app/src/main/java/com/mariotti/developer/futureclock/models/database/AlarmDbSchema.java
package com.mariotti.developer.futureclock.models.database;

public class AlarmDbSchema {
    public static final class AlarmTable {
        public static final String NAME = "alarms";

        public static final class Cols {
            public static final String UUID = "uuid";
            public static final String TIME = "time";
            
            public static final String MONDAY = "monday";
            public static final String TUESDAY = "tuesday";
            public static final String WEDNESDAY = "wednesday";
            public static final String THURSDAY = "thursday";
            public static final String FRIDAY = "friday";
            public static final String SATURDAY = "saturday";
            public static final String SUNDAY = "sunday";

            private static final String TIMEZONE = "timezone";
            public static final String ACTIVE = "active";
        }
    }
}

File: app/src/main/java/com/mariotti/developer/futureclock/models/database/AlarmDbSchema.kt
package com.mariotti.developer.futureclock.models.database

class AlarmDbSchema {
    object AlarmTable {
        val NAME = "alarms"

        object Cols {
            val UUID = "uuid"
            val TIME = "time"

            val MONDAY = "monday"
            val TUESDAY = "tuesday"
            val WEDNESDAY = "wednesday"
            val THURSDAY = "thursday"
            val FRIDAY = "friday"
            val SATURDAY = "saturday"
            val SUNDAY = "sunday"

            val TIMEZONE = "timezone"
            val ACTIVE = "active"
        }
    }
}